config/locales/devise.en.yml

```diff
@@ -23,7 +23,7 @@ en-GB:
         subject: "Please confirm your new password request"
         hello: "Dear %{name}"
         someone_requested_reset: "We received a request to change your password."
-        guide: "You can follow this link to "
+        guide: "To do this, copy and paste the link below into a firefox browser window: "
```

Comment:
Maybe capitalise the word 'firefox' ? Ask @leighmoney to confirm.

The only reason I mention this is because the email will be all text so it would be good to make it obvious to use Firefox.

Or maybe even add **"(and not Internet Explorer)!"**